tests/test_report.py

```diff
@@ -30,103 +31,49 @@
                 "<br>",
                 "<h2>Plots</h2>",
             ],
-        ),
-        (
-            [("everything is fine", {Level.INFO: [("summary",)]})],
-            ["<h2>Details</h2>", "<h2>Plots</h2>"],
-        ),
+        )
     ],
 )
-def test_write_details(mocker, get_df, capsys, messages, expected_details):
-    mock_pio_show = mocker.patch("plotly.io.show", autospec=True)
-    md_mock = mocker.patch("arche.report.display_markdown", autospec=True)
+def test_display(mocker, get_df, capsys, messages, expected_details):
+    mocked_display = mocker.patch("arche.report.display_html", autospec=True)
 
     r = Report()
     for m in messages:
         result = create_result(*m, stats=[get_df])
         r.save(result)
-    r.write_details()
-    mock_pio_show.assert_called_with(result.figures[0])
-    calls = [mocker.call(e) for e in expected_details]
-    md_mock.assert_has_calls(calls, any_order=True)
+    r()
+
+    generated_html = mocked_display.mock_calls[0][1][0]
+    assert generated_html.count("Plotly.newPlot") == 2
+    assert generated_html.count("rule name here - SKIPPED") == 2
+    assert generated_html.count("other result there - SKIPPED") == 2
 
 
 @pytest.mark.parametrize(
     "message, expected_details",
     [({Level.INFO: [("summary", "very detailed message")]}, "very detailed message")],
 )
-def test_write_rule_details(capsys, message, expected_details):
+def test_write_rule_details(mocker, get_df, capsys, message, expected_details):
```

Comment:
```suggestion
def test_report_call_argument(mocker, get_df, capsys, message, expected_details):
```
Would be nice if we remove `@pytest.mark.parametrize(` since we have only one set.